src/app/styles/common.ts

```diff
@@ -8,6 +8,10 @@ export default css`
     height: 100%;
   }
 
+  p {
+    line-height: 1.5;
+  }
+
```

Comment:
Hva er enheten her? `rem`, `em`, `px`, annet?

Reply:
Godt spørsmål! 👍
**Unitless line heights**
The recommended method for defining line height is using a number value, referred to as a “unitless” line height. A number value can be any number, including a decimal-based number, as is used in the first code example on this page.

Unitless line heights are recommended due to the fact that child elements will inherit the raw number value, rather than the computed value. With this, child elements can compute their line heights based on their computed font size, rather than inheriting an arbitrary value from a parent that is more likely to need overriding.

For mer info: https://css-tricks.com/almanac/properties/l/line-height/